### PR TITLE
Fix bugs in SugCmdParsers

### LINKED ISSUE #417: Fix bugs in SugCmdParser
1. mistype in the invalid open path response text
2. `new`  commands without prefix `-t` is an invalid command
3. if you move from new to new - to new -t, you'll see a weird response progression.

### PR DESCRIPTION
**Pull request by johannagwan**
Closes #417

### CODE REVIEW

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java [hunk 1]
```diff
@@ -38,12 +43,22 @@ public NewSuggestionCommandParser(Model model) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(userInput, PREFIX_TITLE, PREFIX_BODY, PREFIX_JUMP);
```
Comment (kevinputera) [inline src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Should we remove `PREFIX_BODY`?

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java [hunk 2]
```diff
@@ -22,6 +23,10 @@
     private static final String RESPONSE_MESSAGE = "Create a new note";
     private static final String RESPONSE_MESSAGE_WITH_TITLE = "Create a new note titled \"%s\".";
     private static final String RESPONSE_MESSAGE_WITH_TITLE_AND_OPEN = "Create a new note titled \"%s\" and open it.";
+    private static final String ERROR_MESSAGE = "\"%s\" is an invalid command format. "
```
Comment (kevinputera) [inline src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Sorry nitpicking here. `"\"%s\" is an invalid creation format. "` better?

File: src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
     private static final String RESPONSE_MESSAGE = "Open a note";
     private static final String RESPONSE_MESSAGE_WITH_TITLE = "Open a note titled \"%s\"";
-    private static final String RESPONSE_MESSAGE_CANNOT_OPEN_NOTE = "Open delete \"%s\". Invalid path.";
+    private static final String RESPONSE_MESSAGE_CANNOT_OPEN_NOTE = "\"%s\" is an invalid path";
```
Comment (kevinputera) [inline src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java]: Should we phrase this as `"Cannot open \"%s\" as it is an invalid path"`?

Also note that your suggestion text for the delete command is inconsistent with this!

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java [hunk 4]
```diff
@@ -38,12 +43,22 @@ public NewSuggestionCommandParser(Model model) {
...
+            model.setResponseText(String.format(ERROR_MESSAGE, model.getInput()));
+            return Optional.empty();
+        }
+
+        String title = argMultimap.getValue(PREFIX_TITLE).get();
+
+        if (!Title.isValidTitle(title) && !title.isBlank()) {
+            model.setResponseText(String.format(ERROR_MESSAGE_INVALID_TITLE, title));
+            return Optional.empty();
         }
```
Comment (kevinputera) [inline src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Hmm why not combine this with the `if else` group below?

**Review by kevinputera — APPROVED** [review]
LGTM, some comments below

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java (hunk 1 above)
Comment (johannagwan) [reply src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Woops sorry I missed it!

Comment (johannagwan) [reply src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: I was blind

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java (hunk 2 above)
Comment (johannagwan) [reply src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Ok!

File: src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java (hunk 4 above)
Comment (johannagwan) [reply src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java]: Ok!

File: src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java (hunk 3 above)
Comment (johannagwan) [reply src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java]: Ok!

**Pull request merged**